pkg/asset/agent/image/ignition.go

```diff
@@ -110,8 +110,14 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 	}
 	logrus.Infof("The rendezvous host IP (node0 IP) is %s", nodeZeroIP)
 
-	// TODO: don't hard-code target arch
-	releaseImageList, err := releaseImageList(agentManifests.ClusterImageSet.Spec.ReleaseImage, "x86_64")
+	rhcosArch := infraEnv.Spec.CpuArchitecture
+	switch rhcosArch {
+	case "arm64":
+		rhcosArch = "aarch64"
+	case "amd64", "":
+		rhcosArch = "x86_64"
```

Comment:
can we do ppc64le at the same time?

Reply:
We don't need to  do this translation with ppc64le (or s390x) because`$GOARCH` and `uname -m` are the same